WebAppConfig.ps1

```diff
@@ -0,0 +1,67 @@
+param(
+	[string]$webAppTarget,
```

Comment:
PowerShell parameter names should be PascalCase

Reply:
Will update that...